src/nvim/mouse.c

```diff
@@ -1133,6 +1122,24 @@ bool is_mouse_key(int c)
          || c == K_X2RELEASE;
 }
 
+static void do_tabclose(int c1)
```

Comment:
This function name is confusing as there is already a `:tabclose` Ex command. I think the function name should contain the word "mouse". Its position in the file is also strange. I think it should be after `move_tab_to_mouse()`.

Reply:
Agree, would `mouse_tab_close()`  be more suitable?

Reply:
Yes, sounds good.